src/epic/web/index.js

```diff
@@ -0,0 +1,418 @@
+                'MW',
+                'MX',
+                'MY',
+                'MZ',
+                'NA',
+                'NC',
+                'NE',
+                'NG',
+                'NI',
+                'NP',
+                'NU',
+                'OM',
+                'PA',
+                'PE',
+                'PG',
+                'PH',
+                'PK',
+                'PN',
+                'PR',
+                'PS',
+                'PW',
+                'PY',
+                'QA',
+                'RU',
+                'RW',
+                'SA',
+                'SB',
+                'SC',
+                'SD',
+                'SG',
+                'SL',
+                'SN',
+                'SO',
+                'SR',
+                'SS',
+                'ST',
+                'SV',
+                'SX',
+                'SY',
+                'SZ',
+                'TC',
+                'TD',
+                'TG',
+                'TH',
+                'TJ',
+                'TK',
+                'TL',
+                'TM',
+                'TN',
+                'TO',
+                'TT',
+                'TW',
+                'TZ',
+                'UA',
+                'UG',
+                'UM',
+                'UY',
+                'UZ',
+                'VC',
+                'VE',
+                'VG',
+                'VI',
+                'VN',
+                'VU',
+                'WS',
+                'YE',
+                'ZA',
+                'ZM',
+                'ZW',
+            ],
+            supportInternationalisationId: 'int',
+        },
+        NZDCountries: {
+            name: 'New Zealand',
+            currency: 'NZD',
+            countries: ['NZ', 'CK'],
+            supportInternationalisationId: 'nz',
+        },
+        Canada: {
+            name: 'Canada',
+            currency: 'CAD',
+            countries: ['CA'],
+            supportInternationalisationId: 'ca',
+        },
+    };
+
+    let countryGroup;
+    const availableCountryGroups = Object.keys(countryGroups);
+    availableCountryGroups.forEach(cg => {
+        if (countryGroups[cg].countries.includes(countryCode)) {
+            countryGroup = cg;
+        }
+    });
+    if (!countryGroup) {
+        return;
+    }
+
+    const extendedCurrencySymbol = {
+        GBPCountries: '£',
+        UnitedStates: '$',
+        AUDCountries: '$',
+        Canada: 'CA$',
+        EURCountries: '€',
+        NZDCountries: 'NZ$',
+        International: '$',
+    };
+
+    return extendedCurrencySymbol[countryGroup];
+}
+
+function changeCurrencySymbolBasedOnLocation() {
+    const countryCode = getDataAttribute('country-code');
+    if (!countryCode) {
+        return;
+    }
+
+    const currency = getCurrencySymbolForCountryCode(countryCode);
+    if (!currency) {
+        return;
+    }
+
+    const elements = document.getElementsByClassName('js-currency-symbol');
```

Comment:
I'm pretty sure this will do what you need:

```javascript
const currencySymbol = document.querySelector('.js-currency-symbol');
if (currencySymbol) {
  currencySymbol.innerHTML = currency;
}
```

Reply:
tres nice